### DISCUSSION TITLE
Add more video cards

### DISCUSSION THREAD

**Discussion by lazerode** [86Box/86Box#3680] [Ideas]
List of video cards to add:
- [ ] [Video 7 VRAM II](http://www.dosdays.co.uk/topics/Manufacturers/video7.php) (not the same as HT208)
- [ ] Compaq AVGA
- [ ] ATI Rage (original version)
- [ ] ATI Rage II
- [ ] S3 Aurora64V+

**Comment by pankoza2-pl** [comment]
the Voodoo 5500 is out of the scope for 86Box, and only the very first ATi Rage is in scope, but the others can be possibly implemented if we find their VBIOSes

**Comment by DDXofficial** [comment]
I am no developer, but I have witnessed enough developer discourse to realize that adding video cards to this project is NO easy task. It does not involve just implementing the driver or any relatively easy-to-find file (such as a driver) to the project; it is unfortunately more complicated than that.

First, there needs to be some kind of documentation about the card (i.e. data sheets with extremely technical information such as schematics, registers, etc.) and BIOS files; even then, there is no guarantee that the video card can be properly emulated due to several extraneous factors such as other lines of code potentially interfering with the card's operation.

You, unfortunately, have provided ZERO documentation for any cards you listed above - I checked the link you posted for the VRAM II and it does not contain enough information for us to begin work on adding that video card (no data sheets, no BIOS files, nothing of the sort) to the emulator. Additionally, NOT ONE of the other cards you listed above has any info alongside it. How could you just request a list of video cards to be added to the project without even making the effort to provide any supporting documentation that could actually aid in emulating said video card?

Please do not waste our time by making such simplistic "list of video cards" requests without providing any other info again. 
Thank you.

**Comment by pankoza2-pl** [comment]
also, Radius SVGA Multiview uses the same HT209 SVGA Chipset as Video 7 VRAM II, except the Radius card has some additional circuitry to support their proprietary Radius Pivot monitor
but good luck finding the VBIOS for the Video 7 VRAM II Card
